### PR TITLE
CC-528,  adding in staging and Live variables.

### PR DESCRIPTION
**Pull request by rwalrondch**

### CODE REVIEW

File: terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars [hunk 1]
```diff
@@ -0,0 +1,23 @@
+environment = "live"
+aws_profile = "live-eu-west-2"
+
+account_local_url = "https://account.cidev.aws.chdev.org"
+ap01_web = "false"
+api_url = "https://api.cidev.aws.chdev.org"
```
Comment (tobafolaranmi-ch) [inline terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars:6]: https://github.com/companieshouse/officer-filing-web/pull/130/files#r1342874023

File: terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
+chs_url = "https://cidev.aws.chdev.org"
+cookie_domain = "cidev.aws.chdev.org"
```
Comment (tobafolaranmi-ch) [inline terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars:9]: https://github.com/companieshouse/officer-filing-web/pull/130/files#r1342874023

File: terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars [hunk 3]
```diff
@@ -0,0 +1,23 @@
+environment = "staging"
+aws_profile = "staging-eu-west-2"
+
+account_local_url = "https://account.cidev.aws.chdev.org"
```
Comment (tobafolaranmi-ch) [inline terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars:4]: https://github.com/companieshouse/officer-filing-web/pull/130/files#r1342874023

File: terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+ap01_web = "true"
+api_url = "https://api.cidev.aws.chdev.org"
+cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
+chs_url = "https://cidev.aws.chdev.org"
```
Comment (tobafolaranmi-ch) [inline terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars:8]: https://github.com/companieshouse/officer-filing-web/pull/130/files#r1342874023

File: terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars [hunk 5]
```diff
@@ -0,0 +1,23 @@
+environment = "staging"
+aws_profile = "staging-eu-west-2"
+
+account_local_url = "https://account.cidev.aws.chdev.org"
+ap01_web = "true"
+api_url = "https://api.cidev.aws.chdev.org"
+cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
```
Comment (tobafolaranmi-ch) [inline terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars:7]: https://github.com/companieshouse/officer-filing-web/pull/130/files#r1342874023

File: terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars [hunk 6]
```diff
@@ -0,0 +1,23 @@
+environment = "live"
+aws_profile = "live-eu-west-2"
+
+account_local_url = "https://account.cidev.aws.chdev.org"
```
[the remark this answers is not in the document]
Comment (rwalrondch) [reply terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars:4]: its ok i should be able to copy another web vars.

**Review by tobafolaranmi-ch — APPROVED** [review]
lgtm

**Pull request merged**